### PR TITLE
add flag --skip-sbom to package create to speed up dev packages

### PR DESCRIPTION
**Pull request by jeff-mccoy**

### CODE REVIEW

File: src/internal/sbom/catalog.go [hunk 1]
```diff
@@ -24,6 +25,12 @@ var viewerAssets embed.FS
 var tranformRegex = regexp.MustCompile(`(?m)[^a-zA-Z0-9\.\-]`)
 
 func CatalogImages(tagToImage map[name.Tag]v1.Image, sbomDir, tarPath string) {
+	// Ignore SBOM creation if there the flag is set
+	if config.SkipSBOM {
```
Comment (YrrepNoj) [inline src/internal/sbom/catalog.go:29]: I personally prefer the pattern of checking this before the function is called. It's all personal preference though so I'm just talking to talk right now.

**Review by YrrepNoj — APPROVED** [review]
Nice QoL change!

**Comment by jeff-mccoy** [comment]
Here for u devz 🦄

File: src/internal/sbom/catalog.go (hunk 1 above)
Comment (jeff-mccoy) [reply src/internal/sbom/catalog.go:29]: I did that at first but it was an extra line of code and I'm evil 😭.  I don't mind if you want to change it though.

**Pull request merged**